official/resnet/cifar10_test.py

```diff
@@ -110,14 +109,6 @@ def test_cifar10_model_fn_train_mode_v1(self):
   def test_cifar10_model_fn_trainmode__v2(self):
     self.cifar10_model_fn_helper(tf.estimator.ModeKeys.TRAIN, version=2)
 
-  def test_cifar10_model_fn_train_mode_multi_gpu_v1(self):
```

Comment:
could we instead change these tests to run distributed strategy version on multiple GPUs?